integration_tests/integration/choosePrisoner.spec.ts

```diff
@@ -89,4 +90,19 @@ context('SignIn', () => {
         expect(altText).equal('Headshot of Stanton, Harry')
       })
   })
+
+  it('Only the names and images of prisoners arriving from court and without a prisoner number will be clickable', () => {
+    cy.signIn()
+    const choosePrisonerPage = Page.verifyOnPage(ChoosePrisonerPage)
+
+    choosePrisonerPage.prisonerImageLink().should('have.length', 1)
+    choosePrisonerPage.prisonerImageLink().eq(0).click()
+    Page.verifyOnPage(ConfirmArrivalPage)
+
+    cy.go('back')
+
+    choosePrisonerPage.prisonerNameLink().should('have.length', 1)
```

Comment:
meh.. by specifying `prisonerNameLink` we're binding to the specific structure of the page. The tests don't really need to know that the action is associated with where the prisoners name is or that it is a link. 

We can avoid this by jumping to a higher layer of abstraction and call it `confirmArrival`

The benefit of this is that the tests are more decoupled so less susceptible from changes in design.

We could represent this control as a massive button to the right of the row and not update the tests.